specification/maps/data-plane/DataRegistry/preview/2022-12-01-preview/examples/DataRegistries_RegisterOrReplace.json

```diff
@@ -0,0 +1,68 @@
+{
+  "parameters": {
+    "geography": "us",
+    "api-version": "2022-12-01-preview",
+    "udid": "29a92cab-74af-49e7-a326-66898220aa9d",
+    "dataRegistry": {
+      "description": "Contoso Geofence GeoJSON",
+      "dataFormat": "geojson",
+      "connection": {
+        "type": "AzureBlobFile",
+        "msiClientId": "3263cad5-ed8b-4829-b72b-3d1ba556e373",
+        "linkedResource": "my-storage-account",
+        "scope": {
+          "containerName": "my-container",
+          "blobPath": "/my/blob/path.json"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Operation-Id": "0522b4fe-0539-4c6c-ba70-e35ec865d8e2",
+        "Operation-Location": "https://us.atlas.microsoft.com/mapData/operations/0522b4fe-0539-4c6c-ba70-e35ec865d8e2"
```

Comment:
https://us.atlas.microsoft.com/mapData/operations/0522b4fe-0539-4c6c-ba70-e35ec865d8e2
should be xxx/dataRegistries/operations/xxx